plasma_framework/contracts/src/exits/payment/PaymentInFlightExitModelLib.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "../../utils/Bits.sol";
+import { PaymentExitDataModel as ExitModel } from "./PaymentExitDataModel.sol";
+
+library PaymentInFlightExitModelLib {
```

Comment:
I wonder about more descriptive name to show its purpose...
`PaymentIFEModelUtils`, `PaymentIFEModelCommon`, `PaymentIFEModelHelper` ...